core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -594,9 +568,9 @@ protected void hideTabSwitcher() {
         tabSwitcherRoot.setVisibility(View.GONE);
         startAnimation(tabSwitcherRoot, R.anim.slide_up);
         progressBar.setVisibility(View.VISIBLE);
-        progressBar.hide();
```

Comment:
this doesn't seem like it should be moved?

Reply:
When exiting the tab view the progress bar remains visible unless this is moved. This is the case for the develop branch as well.

Reply:
grand